test/mpi/maint/test_coll_algos.sh

```diff
@@ -201,11 +201,14 @@ for algo_name in ${algo_names}; do
         echo "allred2 4 arg=-evenmemtype=host arg=-oddmemtype=device ${env}" >> ${testlist_gpu}
         echo "allred2 4 arg=-evenmemtype=reg_host arg=-oddmemtype=device ${env}" >> ${testlist_gpu}
         echo "allred2 4 arg=-evenmemtype=device arg=-oddmemtype=device ${env}" >> ${testlist_gpu}
-        echo "allred3 10 ${env}" >> ${testlist_cvar}
-        echo "allred4 4 ${env}" >> ${testlist_cvar}
         echo "allred5 5 ${env}" >> ${testlist_cvar}
-        echo "allred6 4 ${env}" >> ${testlist_cvar}
-        echo "allred6 7 ${env}" >> ${testlist_cvar}
+        if false ; then
```

Comment:
Just delete these? We could leave the comment, but not sure we need the `if false` block.